src/System.Private.Uri/src/System.Private.Uri.builds

```diff
@@ -10,14 +10,6 @@
     </Project>
     <Project Include="System.Private.Uri.csproj">
       <OSGroup>Windows_NT</OSGroup>
-      <TargetGroup>netstandard13aot</TargetGroup>
-    </Project>
-    <Project Include="System.Private.Uri.csproj">
```

Comment:
oh I see, I thought netstandard13aot was already contained in a shipped stable package so that's why I removed the x-compilation for it. If that's not the case, then I'll add that back.